CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contribution Guide For Developers
+
+## How To Contribute
+
+Thank you for your interest in contributing. As we aim to make it as easy as possible to contribute changes, his guide will introduce basic guidelines on code, documentation and tests that contributors must follow to ensure contributions are accepted quickly.
+
+## Pull Request
+
+The changes should be submitted through 'Pull Request'. Don't send your patches through other channels like email or slack. The submitted PR will be reviewed by the other developers and maintainers and it will be merged by 'Squash Merge'.
+
+We introduce several simple rules for your PR.
+
+* Follow the [7 rules of a great commit message](https://chris.beams.io/posts/git-commit/)
```

Comment:
One of those rules is to capitalize your commit message ;)

Reply:
ㅋㅋㅋㅋ @outsideris You break the new rules 1st. :)

Reply:
Haha. I got it.